conference/src/main/java/com/kyriba/conference/management/domain/Hall.java

```diff
@@ -1,18 +1,24 @@
 package com.kyriba.conference.management.domain;
 
 
-import lombok.EqualsAndHashCode;
-import lombok.Getter;
-import lombok.ToString;
+import lombok.Data;
+import lombok.NoArgsConstructor;
 
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
 
-//@Entity
-@Getter
-@ToString
-@EqualsAndHashCode
+
+@Entity
+@Data
+@NoArgsConstructor
 public class Hall
 {
+  @Id
+  @GeneratedValue
   private Long id;
+  @Column(length = 100, unique = true, nullable = false)
```

Comment:
Useless annotation if schema is not generated from mapping

Reply:
Can't we apply validation that compares DB schema and these rules?

Reply:
Validation doesn't check for constraints, not sure about column length. But managing 2 schema definitions (one for migration and another one for validation) is overhead.